### DISCUSSION TITLE
Incremental Static Regeneration Not Working

### DISCUSSION THREAD

**Discussion by shaffy9633** [vercel/community#1656] [Help]
### Summary

Hi, we are currently facing an issue where our ISR is not regenerating the site every 5 seconds. This is causing data to be not fetched from our backend. Can someone please give some direction on why this is occurring and any leads on how to fix it.

**Comment by shaffy9633** [comment]
<img width="1047" alt="ISR Issue" src="https://user-images.githubusercontent.com/110626159/221994203-1dbd2468-45e5-48ad-b3c9-4bc504d21b19.png">
this is happening only during build time and not when the site is live

**Comment by shaffy9633** [comment]
@amyegan @swarnava @TooTallNate @leerob @EndangeredMassa

**Comment by amyegan** [comment]
Hi @shaffy9633. Do you have an open source repo or a minimal reproduction that we can use to see the problem in action?

**Reply by shaffy9633** [reply]
@amyegan we only have closed source repo with a pro account. We can share it with you if you want :). Our project Id: BluMint . Our team name vercel.com/blumint

**Reply by swarnava** [reply]
Hi,

Looks like you already have a case with us as well. Did you try upgrading to latest Next.js version and see if it helps?

**Reply by shaffy9633** [reply]
@swarnava Hi, we are still amidst transitioning into Next 13. But that will only be up in the next two weeks. Is it possible for me to call someone and troubleshoot this issue. It is rather urgent. Im willing to pay.

**Reply by shaffy9633** [reply]
@amyegan @swarnava is anyone still able to help me on this?

**Reply by amyegan** [reply]
Hi @shaffy9633. It looks like Swarnava was working with Joe last week on the case he mentioned. But I'm happy to work with you here if you prefer that!

Can you tell me more about the situation? For example, are you seeing error messages or other unintended behavior as a sign that ISR isn't working? Please share any reproduction steps, example deployment links, or other information that you're comfortable sharing in public (or update the case with these details if you prefer to keep it private).

**Reply by shaffy9633** [reply]
okay. @amyegan are you able to look at our vercel profile from your system? So basically, our about 1.5 weeks ago, our ISR just stopped working. There isnt much logging from where we can see why it stopped. However, whenever we are redeploying the site, it shows that ISR is working on Build. Is there a way to visually check the  status of ISR without having to redeploy? https://vercel.com/blumint/agora this is our project name. I was hoping you will be able to look into it from your system. If not, we would also be happy to invite you into our project to check any config that we may need
